content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md

```diff
@@ -0,0 +1,186 @@
+the subproject discussed its goal for cleaning up the rest of periodic tests
+and ensuring they passed without flakes.
+
+Elana Hashman joined the subproject, stepping up to help lead it after
+Victor's departure.
+
+***February 2021.*** Artyom Lukianov becomes a Kubernetes org member for his
+contributions to the subproject.
+
+***August 2021.*** After SIG Node successfully ran a [bug scrub](https://groups.google.com/g/kubernetes-dev/c/w2ghO4ihje0/m/VeEql1LJBAAJ)
+to clean up its bug backlog, the scope of the meeting was extended to
+include bug triage to increase the k8s overall reliability, anticipating issues
+before they affect CI signal.
+
+Subproject leads Elana Hashman and Sergey Kanzhelev are both recognized as
+approvers on all node test code, supported by SIG Node and SIG Testing.
+
+***September 2021.*** After significant deflaking progress with serial tests in
+the 1.22 release spearheaded by Francesco Romani, the subproject set a goal
+for getting the serial job fully passing by the 1.23 release date.
+
+Mike Miranda becomes a Kubernetes org member for his contributions
+to the subproject.
+
+***November 2021.*** Over the course of 2021, Node had no merge or release
+blocking test failures. Many flaky tests from past releases are removed from
+release blocking dashboards as they had been fully cleaned up.
+
+Danielle Lancashire was recognized as a reviewer for SIG Node test code.
+
+The final node serial tests were completely fixed. The serial tests consist of
+many disruptive and slow tests which have a tendency to flake and are hard
+to troubleshoot. By the 1.23 release freeze, the last serial tests were
+fixed and the job was passing without flakes.
+
+[![Slack announcement that Serial tests are green](serial-tests-green.png)](https://kubernetes.slack.com/archives/C0BP8PW9G/p1638211041322900)
+
+The 1.23 release got a special shout out for the tests quality and CI signal.
+The SIG Node CI subproject was proud to have helped contribute to such
+a high quality release, in part due to our efforts in identifying
+and fixing flakes in Node and beyond.
+
+[![Slack shoutout that release was mostly green](release-mostly-green.png)](https://kubernetes.slack.com/archives/C92G08FGD/p1637175755023200)
+
+***December 2021.*** An estimated 90% of test jobs were passing at the time of
+the 1.23 release (up from 42% in August 2020).
+
+Dockershim was removed from Kubernetes. This affected nearly half of SIG Node's
+test jobs and the SIG Node CI subproject reacted quickly and retargeted all the
+tests. SIG Node was the first SIG to complete test migrations off dockershim,
+providing examples for other affected SIGs. The vast majority of new jobs passed
+at the time of introduction without further fixes required. There are still
+some wrinkles from the dockershim removal as we uncover more dependencies on
+dockershim, but we plan to stabilize all test jobs by the 1.24 release.
```

Comment:
The removal is contentious even now. Please link to `/dockershim` (or, for the contributor site, to https://k8s.io/dockershim) with some text to let people know they can read more about that.

Reply:
done